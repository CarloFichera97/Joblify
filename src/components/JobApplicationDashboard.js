import React from "react";

const JobApplicationDashboard = () => <div> This is from the Dashboard</div>;

export default JobApplicationDashboard;
