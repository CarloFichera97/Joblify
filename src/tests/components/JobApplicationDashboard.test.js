import React from "react";
import JobApplicationDashboard from "../../components/JobApplicationDashboard";
import { shallow } from "enzyme";

test("Should render Job Application Dashboard correctly", () => {
  const wrapper = shallow(<JobApplicationDashboard />);
  expect(wrapper).toMatchSnapshot();
});
