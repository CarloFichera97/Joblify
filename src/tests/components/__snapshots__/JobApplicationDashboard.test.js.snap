// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Job Application Dashboard correctly 1`] = `
<div>
   
  <Connect(FilterSelector) />
  <Connect(JobApplicationList) />
   
</div>
`;
